frontend_lib/src/appContentFactory.js

```diff
@@ -414,6 +417,81 @@ export function appContentFactory (WrappedComponent) {
       translationState: initialCommentTranslationState
     })
 
+    addContentToFavoriteList = async (content, loggedUser, setState) => {
+      const response = await addContentToFavoriteList(
+        this.apiUrl,
+        loggedUser.userId,
+        content.content_id
+      )
+      if (!response.ok) {
+        GLOBAL_dispatchEvent({
+          type: CUSTOM_EVENT.ADD_FLASH_MSG,
+          data: {
+            msg: i18n.t('Error while adding content to favorites'),
+            type: 'warning',
+            delay: undefined
+          }
+        })
```

Comment:
Use `sendGlobalFlashMessage` from `helper.js`. See also #3941

Reply:
I wasn't aware it existed, will change it.

Reply:
Changed.